integ-test/src/test/java/org/opensearch/sql/legacy/SQLFunctionsIT.java

```diff
@@ -314,6 +314,7 @@ public void castIntFieldToDoubleWithAliasJdbcFormatGroupByTest() {
         rows("35.0"));
   }
 
+  @Ignore("V2 engine doesn't support datetime with TZ")
```

Comment:
You can add new tests instead of disabled. For example:
```sql
SELECT
  CAST(date0 AS date),
  CAST(time0 AS datetime),
  CAST(time1 AS time),  
  CAST(datetime0 AS timestamp),
FROM calcs;
```